plugins/socks5-proxy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: socks5-proxy
+spec:
+  version: "v1.0.0"
+  homepage: https://github.com/yokawasa/kubectl-plugin-socks5-proxy
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/yokawasa/kubectl-plugin-socks5-proxy/archive/v1.0.0.zip
+    sha256: b8d96ab7b7d2ba23d973eeb3a22080829073ae79f8cad5966740fc1ceaf766d3
+    files:
+    - from: "./*/kubectl-socks5-proxy"
+      to: .
+    - from: "./*/LICENSE"
+      to: .
+    bin: "./kubectl-socks5-proxy"
+  shortDescription: A kubectl plugin that creates a local SOCKS5 proxy through which you can access to Services or Pods in a Kubernetes cluster.
+  description: |
```

Comment:
Thanks for already wrapping at 80 characters! :)
And I really like your description 👍